test/image/mocks/box_horz_notched.json

```diff
@@ -0,0 +1,290 @@
+{
+   "data":[
+      {
+         "x":[
+            90,
+            88,
+            55,
+            88,
+            72,
+            100,
+            88,
+            25,
+            92,
+            100,
+            82,
+            82,
+            90,
+            68,
+            85,
+            82,
+            40,
+            100,
+            92,
+            82,
+            55,
+            62,
+            85,
+            100,
+            75,
+            88,
+            78,
+            80,
+            92,
+            100,
+            88,
+            72,
+            95,
+            80,
+            90,
+            72,
+            100,
+            100,
+            75,
+            82,
+            60,
+            90,
+            85,
+            90,
+            38,
+            78,
+            82,
+            100,
+            90,
+            80,
+            80,
+            100,
+            70,
+            100,
+            82,
+            62,
+            92,
+            100,
+            80,
+            100,
+            88,
+            85
+         ],
+         "line":{
+            "color":"#1c9099"
+         },
+         "type":"box",
+         "name":"Notched = True",
+         "orientation": "h",
+         "notched":true
+      },
+      {
+         "x":[
+            70,
+            65,
+            85,
+            75,
+            72,
+            75,
+            90,
+            88,
+            85,
+            80,
+            92,
+            85,
+            85,
+            75,
+            72,
+            80,
+            42,
+            80,
+            95,
+            90,
+            62,
+            65,
+            65,
+            82,
+            68,
+            48,
+            57,
+            95,
+            70,
+            100,
+            80,
+            95,
+            78,
+            80,
+            80,
+            85,
+            90,
+            100,
+            52,
+            85,
+            72,
+            70,
+            45,
+            75,
+            85,
+            95,
+            65,
+            70,
+            85,
+            70,
+            85,
+            35,
+            90,
+            95,
+            95,
+            65,
+            62,
+            48,
+            60,
+            85,
+            85,
+            90,
+            70,
+            68
+         ],
+         "line":{
+            "color":"#1c9099"
+         },
+         "type":"box",
+         "name":"Notch width = 0.1",
+         "notched":true,
+         "orientation": "h",
+         "notchwidth":0.1
```

Comment:
Actually looks like the mocks and baseline images got out of sync. This one uses 0.2 in both images.
https://6171-45646037-gh.circle-artifacts.com/0/home/ubuntu/plotly.js/build/test_images_diff/diff-box_notched.png
https://6171-45646037-gh.circle-artifacts.com/0/home/ubuntu/plotly.js/build/test_images_diff/diff-box_horz_notched.png

Reply:
You are right. I updated the baselines, hope that all will work this time.